### ISSUE TITLE
Improved Accessibility 

### ISSUE THREAD

**Issue by kimwhite** [strangerstudios/pmpro-membership-maps#83]
**Is your feature request related to a problem? Please describe.**
The member map is not WCAG Level 2.1 Level AA compliant because every ARIA button, link and menuitem does not have an accessible name.

**Describe the solution you'd like**
To solve this problem, please fix at least (1) of the following:
1) Element does not have text that is visible to screen readers
2) aria-label attribute does not exist or is empty
3) aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
4) Element has an empty title attribute

**Describe alternatives you've considered**
Other recommendations from our Accessibility Team

**Additional context**
In the state of Colorado, this could result in a fine after July 1, 2024, when a new accessibility law goes into effect.
